.github/workflows/ci.yml

```diff
@@ -20,7 +20,18 @@ jobs:
           allow-prereleases: true
       - name: Install and configure Poetry
         run: |
-          pip install -U pip poetry
-          poetry config virtualenvs.create false
+          if [[ "${{ matrix.python-version }}" == "3.8" ]]; then
```

Comment:
If we are deprecating 3.8 in other PR, there is no point in adding this.

Reply:
Updated. Wait for https://github.com/tortoise/pypika-tortoise/pull/25 to be merged.